lighthouse-core/test/lib/network-request-test.js

```diff
@@ -185,133 +176,5 @@ describe('NetworkRequest', () => {
       expect(record).toMatchObject(req);
       expect(record.lrStatistics).toStrictEqual(undefined);
     });
-
-    it('treats negative timings as 0', function() {
```

Comment:
if we're going to keep the rest of the header code, we should keep these tests as they significantly improved coverage of the code. Would just need to remove (or better, update?) the `startTime`/`endTime`/`responseReceivedTime` numbers

Reply:
We have seperate checks for the timing no-op this is just for negatives, so I removed all the startTime/endTime stuff